test/test_local_io.py

```diff
@@ -42,6 +45,18 @@
     HAS_IOPATH = False
 skipIfNoIoPath = unittest.skipIf(not HAS_IOPATH, "no iopath")
 
+try:
+    import rarfile
+
+    rarfile.tool_setup()
+
+    subprocess.run(("rar", "-?"), check=True)
+
+    HAS_RAR_TOOLS = True
+except (ModuleNotFoundError, rarfile.RarCannotExec, FileNotFoundError, subprocess.CalledProcessError):
+    HAS_RAR_TOOLS = False
```

Comment:
It's interesting that internal CI gets an Error that is not found by GHA. `rarfile` is not found for the exception condition of`RarCannotExec`
```suggestion
try:
    import rarfile
    HAS_RAR_TOOLS = True
    try:
        rarfile.tool_setup()
        subprocess.run(("rar", "-?"), check=True)
    except (rarfile.RarCannotExec, subprocess.CalledProcessError):
        HAS_RAR_TOOLS = False
except (ModuleNotFoundError, FileNotFoundError):
    HAS_RAR_TOOLS = False
```